modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/instance/lifecycle/CommerceServicePortalInstanceLifecycleListener.java

```diff
@@ -161,7 +166,8 @@ private void _addSAPEntries(long companyId, long userId) throws Exception {
 				CommerceOrderService.class.getName(), "#getCommerceOrder\n",
 				CommerceRegionService.class.getName(), "#getCommerceRegions\n",
 				_CLASS_NAME_COMMERCE_CART_RESOURCE, "*\n",
-				_CLASS_NAME_COMMERCE_HEADLESS_CART_RESOURCE, "#getCart\n",
+				_CLASS_NAME_COMMERCE_HEADLESS_CART_RESOURCE, "*\n",
```

Comment:
Do we really need all ?

Reply:
@luca-pellizzon please update this, we can't expose all methods, like delete, getChannelCartsPage and so on, just expose the needed one

Reply:
This PR is not finished, we asked more info in the Jira ticket, in the process to finish it will also fix the permissions really needed.